scripts/scil_sh_to_aodf.py

```diff
@@ -99,7 +117,10 @@ def main():
     args = parser.parse_args()
     logging.getLogger().setLevel(logging.getLevelName(args.verbose))
 
-    if args.use_gpu and args.method == 'cosine':
+    if args.device == 'gpu' and not args.use_opencl:
+        parser.error('Option --use_opencl is required for device \'gpu\'.')
+
+    if args.use_opencl and args.method == 'cosine':
```

Comment:
Is this legacy code? The parser error should say `--use_opencl` I believe.

Reply:
that's right